www/js/lib/uiUtil.js

```diff
@@ -20,27 +20,65 @@
  * along with Kiwix (file LICENSE-GPLv3.txt).  If not, see <http://www.gnu.org/licenses/>
  */
 'use strict';
-define([], function() {
 
+// DEV: Put your RequireJS definition in the rqDef array below, and any function exports in the function parenthesis of the define statement
+// (This method is needed in order to load WebP polyfills conditionally)
```

Comment:
The comment should explain why we load it conditionally : it's not necessary, but an optimization : we want to avoid loading a useless library for the most common use-case (where webp is supported)

Reply:
Done